nfdump: 1.6.16 -> 1.6.17

tobim opened the pull request:
###### Motivation for this change


###### Things done
Also enabled the build of a few additional tools.
Closure size increases from 33.8M to 35.0M.

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

tobim committed:
nfdump: 1.6.16 -> 1.6.17

Build system switched to autotools in the new version.
Also enabled the build of a few additional tools.

markuskowa commented on pkgs/tools/networking/nfdump/default.nix:
```diff
@@ -1,19 +1,33 @@
+  buildInputs = [ bzip2 libpcap ];
+
+  preConfigure = ''
+    patchShebangs autogen.sh
```
That should not be necessary.

markuskowa commented on pkgs/tools/networking/nfdump/default.nix:
```diff
@@ -1,19 +1,33 @@
+
+  preConfigure = ''
+    patchShebangs autogen.sh
+    LIBTOOLIZE=libtoolize ./autogen.sh
```
This should be enough:
```suggestion
    ./autogen.sh
```

tobim replied:
This is just a workaround for darwin. See https://github.com/phaag/nfdump/blob/master/autogen.sh#L5-L7. I can add a condition around it if you wish.

markuskowa replied:
I don't think we need a condition in that case but you could add the comment here stating why it is necessary.

markuskowa commented:
Changelog: https://github.com/phaag/nfdump/releases

tobim committed:
Cleanup

- Remove unneeded call to patchShebangs
- Add a comment explaining the darwin libtoolize workaround
